apply/index.html

```diff
@@ -0,0 +1,76 @@
+			</p>
+				<ol>
+				<li> Which forums are you the most active in?</li>
+				<li>Provide links to 2 – 4 posts you made that you find the most constructive. (NOTE: They must NOT be a Sticky, iTopic, or any kind of guide)</li>
+				<li>Provide a link to your most recent forum post.</li>
+			</ol>
+			<p>Things to keep in mind when submitting your application. Not following these rules could result in your application being declined:</p>
+			<ul>
+				<li>Please do not use any third party service (eg. Ocular) to share deleted posts.</li>
+				<li>Please do not submit posts in your application that were made in the Collaborations, Requests, Project Ideas, Things I'm Making and Creating, Things I'm Reading and Playing, New Scratchers, or Announcements forums.</li>
+				<li>Please do not reply to discussion of your application in the Special Actions studio. You are welcome to read the discussion, but do not reply.</li>
+				<li>Please include the word <span id="codeWord">water</span> in your application to show that you have read and understand this entire page. Neglecting to include this word will cause your application to be auto-rejected.</li>
+				<li>Please do not copy and paste your old application if you are reapplying. We want to see that you have improved since your last application, and reposting the same application does not demonstrate this.</li>
+			</ul>
+
+			<h2>I submitted my application! Now what?</h2>
+			<p>Once you apply, a studio member will check out your application and see if you meet the requirements. If you don't, you will be rejected, in which case you can re-apply in a week. If your application contains all the required components, and you meet the requirements above, a member will post the link in the Forum Helpers Special Actions studio and it will enter a one week "Application Pending" window. During this time, your application will be discussed by current Forum Helpers. If your application gets less than four objections during this time, then your application will be accepted and you will be invited to the studio. If your application gets four or more objections during this time, then your application will be declined and you will be given some feedback on how to improve. If your application is declined, please wait at least one week before reapplying, and try to work on the feedback that you were given before reapplying.</p>
+
+			<h2>I've been accepted! Now what?</h2>
+			<p>Congratulations! You will know that you have been accepted because you will recieve an invite to the studio and a comment on your profile with the good news. Next, please leave a biography (500 characters or less) on the @TFH-Bios account so that it can be added to this website. Welcome to The Forum Helpers!</p>
```

Comment:
Can a link to the TFH-Bios account be added? I think it would be useful.

```suggestion
			<p>Congratulations! You will know that you have been accepted because you will recieve an invite to the studio and a comment on your profile with the good news. Next, please leave a biography (500 characters or less) on the <a href="https://scratch.mit.edu/users/TFH-Bios/" target="_blank">@TFH-Bios</a> account so that it can be added to this website. Welcome to The Forum Helpers!</p>
```

Reply:
how about "if possible"? we don't want to force people to make a bio

Reply:
I think anyone would be able to leave a bio. Maybe something like _"If you wish, please leave a biography..."_

Reply:
I don't think the way it is worded now is forcing anyone to make a bio. If it was worded like "Next, you must submit a bio..." then that could be perceived as hostile, but the way it is worded now already makes it seem optional, but still encourages them to submit one.